src/style.css

```
@@ -0,0 +1,155 @@
+* {
+  box-sizing: border-box;
+}
+
+h3,
+p {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+  --font-family: roboto, sans-serif;
```

try to pass test now
and font-family to body ,it will be inherited by all other element

```suggestion
  font-family: Roboto, sans-serif;
  font-size: 12px;
  font-style: normal;
```